pkgs/servers/http/openresty/default.nix

```diff
@@ -1,25 +1,30 @@
-{ stdenv, fetchurl, fetchgit, openssl, zlib, pcre, libxml2, libxslt, gd, geoip
-, perl }:
-
-assert stdenv.isLinux;
+{ stdenv, fetchurl, fetchgit, openssl, zlib, pcre, postgresql, libxml2, libxslt,
+gd, geoip, perl }:
 
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "openresty-${version}";
-  version = "1.9.3.1";
+  version = "1.13.6.1";
 
   src = fetchurl {
-    url = "http://openresty.org/download/ngx_openresty-${version}.tar.gz";
-    sha256 = "1fw8yxjndf5gsk44l4bsixm270fxv7f5cdiwzq9ps6j3hhgx5kyv";
+    url = "http://openresty.org/download/openresty-${version}.tar.gz";
+    sha256 = "0sv93nw1j597mdpnibmgaa3nnghdk0z6k27vavm8w2c1z9n6w96i";
   };
 
-  buildInputs = [ openssl zlib pcre libxml2 libxslt gd geoip perl ];
+  nativeBuildInputs = [ openssl zlib pcre libxml2 libxslt gd geoip postgresql ];
+  buildInputs = [ perl ];
```

Comment:
Is it possible that you mixed up `nativeBuildInputs` and `buildInputs`. Libraries should be be in `buildInputs` and native tools (the architecture of the host compiling the package) should be in `nativeBuildInputs`.

Reply:
You are right, `nativeBuildInputs` should be in `buildInputs`, they are needed also in runtime. Thanks for fixup.